docs/reports/peer_testing_2/usability_tasks.md

```diff
@@ -0,0 +1,117 @@
+# List of Usability Tasks
+
+## Users
+
+As a user, you are a member of the general public who is using the Kukarella web application; you are someone who would
+like to have a more immersive text-to-speech experience where an animated avatar reads user text aloud.
+
+In your pursuit of having an animated text-to-speech experience, you will use our team's premium Kukarella account, which was provided by the client.
+
+### User Scenario 1: Language Test
+
+1. On the Kukarella web app
+
+    1. Enter text in a language other than English (you may use Google Translate, but your feedback will be more
+    valuable if you are familiar with the language)
+    2. Choose a voice for that language
+    3. Convert your input to a .wav and .txt file
+    4. Inform the examiner which avatar you'd like to use: Barb or Boy
+
+2. Let the examiner do the back-end work that would happen if the API was integrated
+    a. Send the .wav file, .txt file, language, and avatar selection to the API
+    b. Wait for the API to generate the video
+
+3. Watch the video
+
+Repeat this scenario with a different language each time.
+
+### User Scenario 2: Voice Test
+
+1. On the Kukarella web app
+
+    1. Enter text in a language
+    2. Select a voice for that language (try a different one each time you repeat this scenario)
+    3. Convert your input to a .wav and .txt file
+    4. Inform the examiner which avatar you'd like to use: Barb or Boy
+
+2. Let the examiner do the back-end work that would happen if the API was integrated
+    a. Send the .wav file, .txt file, language, and avatar selection to the API
+    b. Wait for the API to generate the video
+
+3. Watch the video
+
+Repeat this scenario with a different voice each time.
```

Comment:
The voices are a kukarella thing so we don't necessarily need to be testing these do we?

Reply:
I'm curious if certain voices that are really deep or high-pitch change the quality of the lip-synching. We can't really do much about it, but it's worth knowing

Reply:
Interesting - yeah a fair thing to test!

Reply:
All daws have basic pitch modification of audio. I can make a couple of these for our next testing session

Reply:
@mkuelker shouldn't be necessary - I think @vnajack discovered kukarella has those features built in